From aa5f3f29babe9b9ecac68290973da18b6e7a264a Mon Sep 17 00:00:00 2001
From: Steven J. Hill <Steven.Hill@imgtec.com>
Date: Thu, 27 Jun 2013 15:27:59 +0000
Subject: [PATCH 030/105] MIPS: SEAD3: Disable L2 cache on SEAD-3.

The cores used on the SEAD-3 platform do not have L2 caches, so
this option should not be turned on. Originally fixed on public
'linux-mti-3.8' release branch.

Signed-off-by: Steven J. Hill <Steven.Hill@imgtec.com>
Cc: linux-mips@linux-mips.org
Patchwork: https://patchwork.linux-mips.org/patch/5559/
Signed-off-by: Ralf Baechle <ralf@linux-mips.org>
(cherry picked from commit 1535ac096d26538888d00881f5026cbc8b6dc20d)
---
 arch/mips/Kconfig |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/arch/mips/Kconfig b/arch/mips/Kconfig
index 97dd298..b6f528f 100644
--- a/arch/mips/Kconfig
+++ b/arch/mips/Kconfig
@@ -341,7 +341,6 @@ config MIPS_SEAD3
 	select DMA_NONCOHERENT
 	select IRQ_CPU
 	select IRQ_GIC
-	select MIPS_CPU_SCACHE
 	select MIPS_MSC
 	select SYS_HAS_CPU_MIPS32_R1
 	select SYS_HAS_CPU_MIPS32_R2
-- 
1.7.1

